src/react/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import ReactDOM from 'react-dom';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Link,
+    useHistory
+} from "react-router-dom";
+
+function App() {
```

Comment:
App is the top-level component that will manage rendering of all pages and components after it. Also, unnecessary div L14